Packs/MicrosoftGraphApplications/Integrations/MicrosoftGraphApplications/MicrosoftGraphApplications.py

```diff
@@ -185,10 +185,12 @@ def remove_password_service_principal(
     def unlock_configuration_service_principal(
```

Comment:
if this does not only unlock (but can also lock), let's change its name to `change_configuration_service_principal_lock_status`

Reply:
Done